bot/cogs/fun.py

```diff
@@ -261,28 +261,29 @@ async def why(self, ctx: Context) -> None:
 
     @command(aliases=["rhash", "robothash", "rh", "rohash"])
     async def robohash(self, ctx: Context, *, meme: str) -> None:
-        """text => robot image thing."""
+        """Text => robot image thing."""
         try:
-            e = Embed(colour=0x690E8)
+            embed = Embed(colour=0x690E8)
             meme = urllib.parse.quote_plus(meme)
-            e.set_image(url=f"https://robohash.org/{meme}.png")
-            await ctx.send(embed=e)
-        except Exception as e:
-            await ctx.send(f"Something Broke. LOL [{e!s}]")
+            embed.set_image(url=f"https://robohash.org/{meme}.png")
+            await ctx.send(embed=embed)
+        except Exception as error:
+            # What is {error!s} ?
+            await ctx.send(f"Something Broke. LOL [{error!s}]")
```

Comment:
Doing `!s` seems to convert the `error` variable to string, but this is already handled by f-strings so it can be safely removed
```suggestion
            await ctx.send(f"An error occurred: {error}")
```

Reply:
Okay. Since I didn't now, and didn't bother to check, I preferred not to edit